<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgPreferences</class>
 <widget class="QDialog" name="dlgPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tbwPreferences">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDataBase">
      <attribute name="title">
       <string>DataBase</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lblDBName">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="ledDBName"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblDBHost">
           <property name="text">
            <string>Host:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="ledDBHost"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblDBUser">
           <property name="text">
            <string>User:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="ledDBUser"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblDBPassword">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="ledDBPassword"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="psbTestConnection">
         <property name="text">
          <string>Test Connection</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLogging">
      <attribute name="title">
       <string>Logging</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="lblConsoleLogLevel">
           <property name="text">
            <string>Console Log Level:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbxConsoleLogLevel"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblFileLogLevel">
           <property name="text">
            <string>File Log Level:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cbxFileLogLevel"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>107</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bbxButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bbxButtons</sender>
   <signal>accepted()</signal>
   <receiver>dlgPreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbxButtons</sender>
   <signal>rejected()</signal>
   <receiver>dlgPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
